components/credentials/src/test/java/org/cloudfoundry/credhub/requests/CertificateGenerationRequestParametersTest.java

```diff
@@ -17,6 +19,9 @@ public class CertificateGenerationRequestParametersTest {
 
   private CertificateGenerationRequestParameters subject;
 
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
```

Comment:
So you decided not to go for the lambda way?

Reply:
Oh, I missed that one. I'll change it.